layer/support/csv_logging.cc

```diff
@@ -17,11 +17,15 @@
 #include <sstream>
 #include <string>
 
-#include "layer/support/debug_logging.h"
 #include "layer/support/event_logging.h"
 #include "layer/support/layer_utils.h"
 
 namespace performancelayers {
+namespace {
+const std::unordered_set<ValueType> kIgnoredEventTypes = {
```

Comment:
I removed the set and just checked the type directly in the filtering step, but I kept the filtering (instead of adding logic to look ahead when adding a delimiter and not adding it if it's `kTraceEvent`). This is basically to cleanly handle cases where the trace event attribute is in the middle or at the start of the attribute list or there are multiple trace events in the list. 

If you think that's never likely to happen, I can just rewrite this assuming that if there's a trace event, it's a single one at the end of the attribute list.